src/Cairo.jl

```diff
@@ -164,20 +165,20 @@ type CairoSurface{T<:Union{UInt32,RGB24,ARGB32}} <: GraphicsDevice
 
     @compat function (::Type{CairoSurface{T}}){T}(ptr::Ptr{Void}, w, h)
         self = new{T}(ptr, w, h)
-        finalizer(self, destroy)
+        finalizer(destroy, self)
```

Comment:
`@compat finalizer(destroy, self)`, see https://github.com/JuliaLang/Compat.jl/pull/422

Reply:
@lobingera this should be fixed before the package os tagged